salesagility/SuiteCRM
# Remove BusinessCard-related code.

# Description
This view doesn't seem to be accessible from anything in the CRM (a search for 'BusinessCard' gives me nothing).

The code references files, like `modules/Notes/NoteFormBase.php`, that don't exist. The only other reference to `NoteFormBase` I can find is in the old change notes from SugarCRM that are from 2008. Presumably, the files were removed a really long time ago, maybe even before Suite was forked. Based on that alone, and the fact that the code hasn't been touched at all (other than via commits that touched hundreds of files, like linter changes or copyright header changes), I think it should be perfectly safe to remove this.

## Motivation and Context

I was trying to replace an old view that was using XTemplate with Smarty, but couldn't figure out how in the world to access the view. Eventually, I figured out how to access the page manually via URL (visit `http://localhost/index.php?action=BusinessCard&module=Contacts&record=CONTACT_RECORD_ID` in your browser) but the code errored because of missing code it references.

This code seems entirely unrelated to the vCard code, in case anyone was thinking that's what this was related to.

## How To Test This
Make sure the tests pass, do a basic sanity check to see if anything breaks as you go through the Contacts module.

## Types of changes
Technical debt.


### Final checklist
